.travis.yml

```diff
@@ -2,8 +2,6 @@ sudo: required
 dist: trusty
 language: generic
 install: git clone https://github.com/QubesOS/qubes-builder ~/qubes-builder
-# debootstrap in trusty is old...
-before_script: sudo ln -s sid /usr/share/debootstrap/scripts/stretch
```

Comment:
It is required to fix the build process, as the change was moved to qubes-builder, as mentioned in the commit message.